php_phongo_classes.h

```diff
@@ -242,18 +242,19 @@ extern zend_class_entry *php_phongo_connectiontimeoutexception_ce;
 extern zend_class_entry *php_phongo_writeexception_ce;
 extern zend_class_entry *php_phongo_bulkwriteexception_ce;
 
-extern zend_class_entry *php_phongo_type_ce;
-extern zend_class_entry *php_phongo_persistable_ce;
-extern zend_class_entry *php_phongo_unserializable_ce;
-extern zend_class_entry *php_phongo_serializable_ce;
 extern zend_class_entry *php_phongo_binary_ce;
 extern zend_class_entry *php_phongo_decimal128_ce;
 extern zend_class_entry *php_phongo_javascript_ce;
 extern zend_class_entry *php_phongo_maxkey_ce;
 extern zend_class_entry *php_phongo_minkey_ce;
 extern zend_class_entry *php_phongo_objectid_ce;
+extern zend_class_entry *php_phongo_persistable_ce;
```

Comment:
The `MongoDB\Driver\Exception` CEs are not alphabetized, but `MongoDB\Driver` and `MongoDB\Driver\Monitoring` CEs were, as were all externs for the init functions. I'll revert this change, though.